lib/Manager.php

```diff
@@ -101,6 +101,10 @@ public function createParticipantObject(Room $room, array $row) {
 	 * @return Room[]
 	 */
 	public function getRoomsForParticipant($participant) {
+		if ($this->config->hideRoomList()) {
```

Comment:
Instead of on the manager, this should be done on the API I think.
Otherwise this will cause problems, because e.g. we can't remove participants from their rooms anymore when the user is deleted etc.

Reply:
Oh, right. Done.